platform/kube/inject/initializer.go

```diff
@@ -111,53 +112,49 @@ func NewInitializer(restConfig *rest.Config, config *Config, cl kubernetes.Inter
 			return nil, err
 		}
 
-		for n := range i.config.IncludeNamespaces {
-			namespace := i.config.IncludeNamespaces[n]
-
-			watchlist := &cache.ListWatch{
-				ListFunc: func(options metav1.ListOptions) (runtime.Object, error) {
-					options.IncludeUninitialized = true
-					options.FieldSelector = fields.Everything().String()
-					return kindClient.Get().
-						Namespace(namespace).
-						Resource(kind.resource).
-						VersionedParams(&options, metav1.ParameterCodec).
-						Do().
-						Get()
-				},
-				WatchFunc: func(options metav1.ListOptions) (watch.Interface, error) {
-					options.IncludeUninitialized = true
-					options.Watch = true
-					options.FieldSelector = fields.Everything().String()
-					return kindClient.Get().
-						Namespace(namespace).
-						Resource(kind.resource).
-						VersionedParams(&options, metav1.ParameterCodec).
-						Watch()
-				},
-			}
-
-			patcher := func(namespace, name string, patchBytes []byte, obj runtime.Object) error {
-				return kindClient.Patch(types.StrategicMergePatchType).
-					Namespace(namespace).
+		watchlist := &cache.ListWatch{
```

Comment:
note: watch all namespaces instead of only included namespaces.

Reply:
In this file, you can use the const IstioNamespace instead of the string "istio-system".